src/styles.css

```diff
@@ -138,7 +141,7 @@ h2 {
   /* border: 2px solid #1B3231; */
   border: 2px solid #F2F5F2;
   height: 300px;
-  width: 275px;
+  width: 273px;
```

Comment:
Allows us to have 4 recipes display on a single 'line' when the page is laptop size.